### PR TITLE
[Breaking] Operationalise up- and download

### PR DESCRIPTION
**Pull request by rjzondervan**
Operationalises up- and downloads of documents into a different entity, and show a link in the original object.

Please note: this breaks compatibility with previous versions.

### CODE REVIEW

**Review by bbrands02 — APPROVED** [review]

File: Service/ZGWService.php [hunk 1]
```diff
@@ -137,5 +144,112 @@ public function drcReleaseHandler(array $data, array $configuration): array
 
         $data['response'] = $object->toArray();
         return $data;
+    }
+
+    private function generateDownloadEndpoint(string $id, Endpoint $downloadEndpoint): string
```
Comment (bbrands02) [inline Service/ZGWService.php:149]: Docblocks?

File: ActionHandler/InhoudHandler.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    /**
+     *  This function returns the requered configuration as a [json-schema](https://json-schema.org/) array.
+     *
+     * @throws array a [json-schema](https://json-schema.org/) that this  action should comply to
+     */
+    public function getConfiguration(): array
+    {
+        return [
+            '$id'         => 'https://vng.opencatalogi.nl/schemas/drc.saveContent.schema.json',
+            '$schema'     => 'https://json-schema.org/draft/2020-12/schema',
```
Comment (WilcoLouwerse) [inline ActionHandler/InhoudHandler.php:26]: '$id'             => 'https://vng.opencatalogi.nl/ActionHandler/drc.InhoudHandler.ActionHandler.json',
'$schema'    => 'https://docs.commongateway.nl/schemas/ActionHandler.schema.json',
Voor consistency

File: Service/ZGWService.php [hunk 3]
```diff
@@ -137,5 +144,112 @@ public function drcReleaseHandler(array $data, array $configuration): array
...
+//            throw new \HttpException('Lock not valid', 400);
+//        }
+
+        $file = $objectEntity->getValueObject('inhoud')->getFiles()->first();
+        $file->setBase64($file->getBase64().$data['inhoud']);
+        $file->setSize(mb_strlen($file->getBase64()));
+
+
+
+
```
Comment (WilcoLouwerse) [inline Service/ZGWService.php:253]: lot of empty lines^
I think this function also needs a return statement here?

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]
Mostly style stuff

**Review by bbrands02 — APPROVED** [review]

**Review by smisidjan — APPROVED** [review]

**Pull request merged**